packages/exporter-trace-otlp-http/src/OTLPExporterBase.ts

```diff
@@ -93,6 +96,11 @@ export abstract class OTLPExporterBase<
 
   private _export(items: ExportItem[]): Promise<unknown> {
     return new Promise<void>((resolve, reject) => {
+      setTimeout(() => {
+        // don't wait anymore for export
+        reject(new Error('Timeout'));
```

Comment:
do this will be logged in the diag level by the sdk ?

Reply:
I'm not that familiar with the diag level logger but I've ran some tests locally with a sample app and when I have this set: 

```
api.diag.setLogger(
  new api.DiagConsoleLogger(),
  api.DiagLogLevel.ERROR
);
```

then I get a `{"stack":"Error: Export Timeout\n    at Timeout._onTimeout `....etc error log in the console. 

Is that what you were asking about?

Reply:
Yes these errors are caught. I think @vmarchaud just wanted to make sure we weren't going to cause a process crash or unhandled promise rejection or anything like that.